index.js

```diff
@@ -1,4 +1,21 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const routes = require('./routers');
+
+const app = express();
+
+const PORT = 3000;
```

Comment:
```suggestion
const PORT = process.env.PORT || 3000;
```
A galera pode usar variáveis de ambiente aqui com o padrão sendo 3000?